modeling-cmds/src/ok_response.rs

```diff
@@ -69,6 +69,13 @@ define_ok_modeling_cmd_response_enum! {
             pub entity_ids: Vec<Uuid>,
         }
 
+        /// The response from the `ClosePath` command.
+        #[derive(Debug, Serialize, Deserialize, JsonSchema, ModelingCmdOutput)]
+        pub struct ClosePath {
+            /// The UUID of the lone face of the resulting solid2D.
+            pub face_id: Option<Uuid>,
```

Comment:
Why is this optional? Under what circumstances would it not have a face ID?

Reply:
If the close operation fails (e.g. a path isn't provided), no ID would be passed back

Reply:
If the close operation fails then the API won't send back any `ClosePath` response at all. So this `face_id` field should be `Uuid` not `Option<Uuid>`.